start.js

```diff
@@ -0,0 +1,39 @@
+const assert = require('assert');
```

Comment:
Its a WIP, this would eventually be test code, but it shows how to call the APIs for now.